### PR TITLE
Add argument in torch trainer whether to predict or not

### PR DESCRIPTION
**Pull request by hjdeheer**
# Adds partial prediction, based on flag ` all_predict` 

# Now also supports numpy arrays as indices

### CODE REVIEW

**Comment by schobbejak** [comment]
Add pr information

**Review by tolgakopar — APPROVED** [review]
Looks good

File: epochalyst/pipeline/model/training/torch_trainer.py [hunk 1]
```diff
@@ -200,9 +201,13 @@ def custom_train(
         concat_dataset: Dataset[Any] = self._concat_datasets(
```
Comment (schobbejak) [inline epochalyst/pipeline/model/training/torch_trainer.py:201]: Should be moved into the if statement otherwise its unnecessary

**Review by schobbejak — CHANGES_REQUESTED** [review]

**Review by schobbejak — DISMISSED** [review]

**Review by schobbejak — APPROVED** [review]
Looks fine

**Pull request merged**